### PR TITLE
New cugraph site structure

### LINKED ISSUE #3297: Create the new index page that captures the structure
create the new page and get the API docs moved over

### PR DESCRIPTION
**Pull request by acostadon**
resolves #3297 

new website structure with some migrations.

### CODE REVIEW

File: docs/cugraph/source/graph_support/PyG_support.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# cugraph_pyg
```
Comment (alexbarghi-nv) [inline docs/cugraph/source/graph_support/PyG_support.md:1]: This document is out of date.  Usage of cugraph-pyg is now significantly different from what is shown in the included example.  In addition, cugraph-service is no longer used by cugraph-pyg; cugraph-pyg now depends exclusively on cugraph.

**Review by alexbarghi-nv — CHANGES_REQUESTED** [review]

**Comment by BradReesWork** [comment]
This will roll out first week of 23.06 release 

changes LGTM

**Comment by acostadon** [comment]
rerun tests

**Review by BradReesWork — APPROVED** [review]

File: docs/cugraph/source/graph_support/PyG_support.md (hunk 1 above)
Comment (acostadon) [reply docs/cugraph/source/graph_support/PyG_support.md:1]: removed that specific info in that page. I will replace it with better text and an example when I update the page in 23.06

File: docs/cugraph/source/graph_support/PyG_support.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+[RAPIDS](https://rapids.ai) cugraph_pyg enables the ability to use cugraph Property Graphs with PyTorch Geometric (PyG).  PyG users will have access to cuGraph through the PyG GraphStore, FeatureStore, and Sampler interfaces.
```
Comment (alexbarghi-nv) [inline docs/cugraph/source/graph_support/PyG_support.md:3]:
```suggestion
[RAPIDS](https://rapids.ai) cugraph_pyg enables the ability to use cugraph graph storage and sampling with PyTorch Geometric (PyG).  PyG users will have access to cuGraph through the PyG GraphStore, FeatureStore, and Sampler interfaces.
```

File: docs/cugraph/source/installation/getting_cugraph.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Conda
+It is easy to install cuGraph using conda. You can get a minimal conda installation with [Miniconda](https://conda.io/miniconda.html) or get the full installation with [Anaconda](https://www.anaconda.com/download).
+
+cuGraph Conda packages
+ * cugraph - this will also import:
+   * pylibcugraph
+   * libcugraph
+ * cugraph_service_client
+ * cugraph_service_server
```
Comment (alexbarghi-nv) [inline docs/cugraph/source/installation/getting_cugraph.md:28]:
```suggestion
 * cugraph-service-server
```

File: docs/cugraph/source/installation/getting_cugraph.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ * cugraph_dgl
```
Comment (alexbarghi-nv) [inline docs/cugraph/source/installation/getting_cugraph.md:29]:
```suggestion
 * cugraph-dgl
```

**Review by alexbarghi-nv — APPROVED** [review]
👍

**Pull request merged**